src/conf.c

```diff
@@ -1599,4 +1599,33 @@ int wolfSSL_CONF_cmd_value_type(WOLFSSL_CONF_CTX *cctx, const char *cmd)
  * END OF CONF API
  ******************************************************************************/
 
+#if defined(WOLFSSL_NGINX)
```

Comment:
I don't like osp specific gating. Put it in ossl all or leave it in extra with no gating.